app/src/main/java/com/example/indoor_volleyball/Activities/LoginActivity.java

```diff
@@ -29,22 +32,12 @@ protected void onCreate(Bundle savedInstanceState) {
         if (ParseUser.getCurrentUser() != null) {
             goMainActivity();
         }
-        //TODO add cut out all my excess code relating to the bindings in both login and sign up.
-        // inflating our xml layout in our activity main binding
         binding = ActivityLoginBinding.inflate(getLayoutInflater());
-
-        // getting our root layout in our view.
-        View view = binding.getRoot();
-
-        // below line is to set
-        // Content view for our layout.
-        setContentView(view);
-
-        //etPassword = binding.etPassword;
-        //etUsername = binding.etUsername;
-
-        // calling button and setting on click listener for our button.
-        // we have called our button with its id and set on click listener on it.
+        setContentView(binding.getRoot());
+        locationPermissionRequest.launch(new String[]{
+                Manifest.permission.ACCESS_FINE_LOCATION,
+                Manifest.permission.ACCESS_COARSE_LOCATION
```

Comment:
Since you use this array twice, maybe save it as a constant so it can be re-used.